### PR TITLE
Do not use buildout to build the doc in travis

### PR DESCRIPTION
**Pull request by guewen**
It halves the time to build the documentation.

### CODE REVIEW

**Comment by guewen** [comment]
Based on https://github.com/OCA/geospatial/pull/31

**Comment by lmignon** [comment]
:+1:

File: .travis_build_doc [hunk 1]
```diff
@@ -1,40 +1,10 @@
...
 cd ${HOME}
-mkdir buildout && cd buildout
-ln -s ${TRAVIS_BUILD_DIR} connector
-ln -s ${HOME}/odoo-${VERSION} odoo
-cat <<EOF > buildout.cfg
-[buildout]
-parts = openerp
-versions = versions
-find-links = http://download.gna.org/pychart/
+mkdir build_doc
```
Comment (pedrobaeza) [inline .travis_build_doc]: Where is this directory used?

Comment (guewen) [inline .travis_build_doc]: Removed

**Comment by bwrsandman** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**